witchcraft-server/Cargo.toml

```diff
@@ -116,6 +107,17 @@ witchcraft-server-config = { version = "5.6.0", path = "../witchcraft-server-con
 witchcraft-server-macros = { version = "5.6.0", path = "../witchcraft-server-macros" }
 zipkin = "1.0"
 
+[target.'cfg(not(windows))'.dependencies]
```

Comment:
tikv-jemallocator doesn't build on windows. The upstream jemalloc project is also archived, so it might be time to think about switching to mimalloc...